doc/classes/CollisionShape3D.xml

```diff
@@ -20,11 +20,11 @@
 				Sets the collision shape's shape to the addition of all its convexed [MeshInstance3D] siblings geometry.
 			</description>
 		</method>
-		<method name="resource_changed">
+		<method name="resource_changed" is_deprecated="true">
 			<return type="void" />
 			<param index="0" name="resource" type="Resource" />
 			<description>
-				If this method exists within a script it will be called whenever the shape resource has been modified.
+				[i]Deprecated.[/i] Use [signal Resource.changed] instead.
```

Comment:
"Deprecated" means that the method is no longer recommended and that an alternative is offered, BUT it should still work. This PR seems to remove the functionality and just keep empty stubs, which is not really useful. Either we deprecated and keep the functionality, or we break compatibility.

Reply:
Alternatively we can keep the stubs to avoid a theoretical API breakage but these should be marked as "obsoleted", not "deprecated". https://stackoverflow.com/a/9208164

Reply:
Changed to "Obsoleted."